python-polars/downloads.py

```diff
@@ -0,0 +1,14 @@
+import requests
+import pathlib
```

Comment:
Minor quibble, but we should put the standard library on top of the imports:

```suggestion
import pathlib
import requests
```